docs/docs/genai/eval-monitor/scorers/llm-judge/prompt.mdx

```diff
@@ -0,0 +1,123 @@
+import { APILink } from "@site/src/components/APILink";
+import ImageBox from "@site/src/components/ImageBox";
+import TileCard from '@site/src/components/TileCard';
+import TilesGrid from '@site/src/components/TilesGrid';
+import { Bot, BookOpen, MessageSquare } from "lucide-react";
+
+# Bring Your Own Prompts
+
+The <APILink fn="mlflow.genai.judges.custom_prompt_judge">custom_prompt_judge</APILink> API is designed to help you quickly and easily create LLM scorers when you need full control over the judge's prompt or need to return multiple output values beyond "pass" / "fail", for example, "great", "ok", "bad".
+
+You provide a prompt template that has placeholders for specific fields in your app's trace and define the output choices the judge can select. The LLM judge model uses these inputs to select the best output choice and provides a rationale for its selection.
+
+:::tip
+
+We recommend starting with <ins>[guidelines-based judges](/genai/eval-monitor/scorers/llm-judge/guidelines)</ins> and only using prompt-based judges if you need more control or can't write your evaluation criteria as pass/fail guidelines. Guidelines-based judges have the distinct advantage of being easy to explain to business stakeholders and can often be directly written by domain experts.
+
+:::
+
+## Example Usage
+
+```python
+from mlflow.genai.judges import custom_prompt_judge
+from mlflow.genai.scorers import scorer
+
+
+issue_resolution_prompt = """
+Evaluate the entire conversation between a customer and an LLM-based agent. Determine if the issue was resolved in the conversation.
+
+You must choose one of the following categories.
+
+[[fully_resolved]]: The response directly and comprehensively addresses the user's question or problem, providing a clear solution or answer. No further immediate action seems required from the user on the same core issue.
```

Comment:
Maybe it's worth calling out only this type of output formatting is supported in this API iirc?